src/components/Footer.tsx

```diff
@@ -1,29 +1,79 @@
 import * as React from 'react';
 import Link from 'gatsby-link';
 
-export default () => (
+interface FooterProps {
+  lng: 'en' | 'hu';
+}
+
+const firstRow = [
+  {
+    en: 'About us',
+    hu: 'Rólunk',
+    link: 'rolunk'
+  },
+  {
+    en: 'Take action',
+    hu: 'Cselekedj',
+    link: 'cselekves'
+  },
+  {
+    en: 'Donate',
+    hu: 'Támogatás',
+    link: 'tamogatas'
+  }
+];
+const secondRow = [
+  {
+    en: 'voluntary',
+    hu: 'önkéntesség',
+    link: 'rolunk#contribution'
+  },
+  {
+    en: 'events',
+    hu: 'események',
+    link: 'sorry'
+  },
+  {
+    en: 'transparency',
+    hu: 'átláthatóság',
+    link: 'sorry'
+  }
+];
+const thirdRow = [
+  {
+    en: 'privacy',
+    hu: 'adatvédelem',
+    link: 'rolunk#privacy'
+  },
+  {
+    en: 'dictionary',
+    hu: 'kisokos',
+    link: 'kisokos'
+  }
+];
+const fourthRow = [
+  {
+    en: 'contributors',
+    hu: 'impresszum',
+    link: 'rolunk#contributors'
+  }
+];
+
+export default (props: FooterProps) => (
   <footer className="general-footer" role="contentinfo">
     <hr></hr>
     <div className="footer-grid-container">
-      <div className="footer-grid-item first-row">
-        <Link to="/rolunk">Rólunk</Link>
-      </div>
-      <div className="footer-grid-item first-row">
-        <Link to="/cselekves">Cselekedj</Link>
-      </div>
-      <div className="footer-grid-item first-row">
-        <Link to="/tamogatas">Támogatás</Link>
-      </div>
+      {firstRow.map(item => (
+        <div className="footer-grid-item first-row" key={item.hu}>
+          <Link to={item.link}>{item[props.lng]}</Link>

```

Comment:
to={`${props.lng}/${item.link}`}
Valamiert megvaltoztattak a <Link>-et es mindenkeppen absolute path-kent ertelmezi.